docs/2.8.0/versions.json

```diff
@@ -1,6 +1,6 @@
 {
-  "daml": "2.8.0-snapshot.20231118.12382.0.v86cb8054",
-  "canton": "2.8.0-snapshot.20231120.11569.0.v2461ac9b",
+  "daml": "2.8.0",
+  "canton": "2.8.0",
   "daml_finance": "1.3.11",
```

Comment:
@johan-da I'm not entirely sure how the docs version matches daml versions for Finance, so I'll leave it up to you to update this line. Feel free to push to this branch with that update.

Reply:
@garyverhaegen-da `1.3.11` is the correct one for `2.8.0`.

Reply:
Thanks for checking!